src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -335,51 +344,244 @@ private synchronized void writeToConsole(Person person, int index, long time, bo
   }
 
   private long setDemographics(Person person, Demographics city) {
-    person.attributes.put(Person.CITY, city.city);
-    person.attributes.put(Person.STATE, city.state);
-    
-    String race = city.pickRace(person.random);
-    person.attributes.put(Person.RACE, race);
-    String ethnicity = city.ethnicityFromRace((String)person.attributes.get(Person.RACE), person);
-    person.attributes.put(Person.ETHNICITY, ethnicity);
-    String language = city.languageFromEthnicity((String) person.attributes.get(Person.ETHNICITY),
-        person);
-    person.attributes.put(Person.FIRST_LANGUAGE, language);
-
-    String gender = city.pickGender(person.random);
-    if (gender.equalsIgnoreCase("male") || gender.equalsIgnoreCase("M")) {
-      gender = "M";
-    } else {
-      gender = "F";
-    }
-    person.attributes.put(Person.GENDER, gender);
-
-    // Socioeconomic variables of education, income, and education are set.
-    String education = city.pickEducation(person.random);
-    person.attributes.put(Person.EDUCATION, education);
-    double educationLevel = city.educationLevel(education, person);
-    person.attributes.put(Person.EDUCATION_LEVEL, educationLevel);
-
-    int income = city.pickIncome(person.random);
-    person.attributes.put(Person.INCOME, income);
-    double incomeLevel = city.incomeLevel(income);
-    person.attributes.put(Person.INCOME_LEVEL, incomeLevel);
-
-    double occupation = person.rand();
-    person.attributes.put(Person.OCCUPATION_LEVEL, occupation);
-
-    double sesScore = city.socioeconomicScore(incomeLevel, educationLevel, occupation);
-    person.attributes.put(Person.SOCIOECONOMIC_SCORE, sesScore);
-    person.attributes.put(Person.SOCIOECONOMIC_CATEGORY, city.socioeconomicCategory(sesScore));
-
-    long targetAge = city.pickAge(person.random);
-
-    // TODO this is terrible date handling, figure out how to use the java time library
-    long earliestBirthdate = stop - TimeUnit.DAYS.toMillis((targetAge + 1) * 365L + 1);
-    long latestBirthdate = stop - TimeUnit.DAYS.toMillis(targetAge * 365L);
-
-    long birthdate = (long) person.rand(earliestBirthdate, latestBirthdate);
-
-    return birthdate;
+	  // Create map and read in the sampled SPEW csv file for Massachusetts 
+
+	  List<LinkedHashMap<String, String>> spewPerson;
+
+	  try {
+		  spewPerson = SimpleCSV.parse(Utilities.readResource("people_25.csv"));
+	  } catch (IOException e) {
+		  e.printStackTrace();
+		  return (Long) null;
```

Comment:
What is the behavior of the program if this actually happens?

E.g. if you delete `people_25.csv` and run the program what happens?

I'm not convinced that swallowing the exception and returning `null` makes sense.

Reply:
If the data's bad then I'd say halt the program. We'll release it with good data, so the only reason it should fail is if the user changes it. My suggestion is let the program crash, let them know why, and let them fix it.

Reply:
Why not just throw an exception then instead of returning null?